.github/workflows/sonarqube.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           [ -f ./setup.sh ] && ./setup.sh || true
       - name: Analyse with Gradle
         run: |
-          ./gradlew check sonarqube --no-daemon --parallel --continue
+          ./gradlew check sonarqube --no-daemon --continue --no-build-cache
```

Comment:
why are we changing this? [Project template has `--parallel --continue`](https://github.com/micronaut-projects/micronaut-project-template/blob/master/.github/workflows/sonarqube.yml#L51)

Reply:
Running functional tests in parallel that create and tear down Kubernetes namespaces is very challenging. For this project, `--parallel` is pointless and useless

Reply:
There are race conditions with creating and reusing k8s namespaces, because of that test cannot be run in parallel and `--no-build-cache` is there because same namespaces creates flaky tests and I wanted to make sure that everything works in same run.